crates/compiler/parse/src/parser.rs

```diff
@@ -2462,7 +2462,7 @@ pub fn one_or_more<'a, Output, E: 'a>(
 macro_rules! debug {
     ($parser:expr) => {
         move |arena, state: $crate::state::State<'a>, min_indent: u32| {
-            dbg!($parser.parse(arena, state, min_indent))
+            dbg!("{:?}", $parser.parse(arena, state, min_indent))
```

Comment:
Did you mean to change this to `eprintln!`?

Reply:
oops - I changed it but then realized it should stay as-is, will revert!